### ISSUE TITLE
Drawing chart within iPhone browser (UIWEBVIEW) throwing error.

### ISSUE THREAD

**Issue by vsoverseas** [DmitryBaranovskiy/g.raphael#31]
[DOMElement structuralComplexityContribution]: unrecognized selector sent to instance 0x3b585d0
2010-04-12 22:01:36.773 Smart Utility[1731:207] \* Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '**\* -[DOMElement structuralComplexityContribution]: unrecognized selector sent to instance 0x3b585d0'

**Comment by leonfu** [comment]
It's a very critical bug that even make UIWebView crash

**Comment by Chralu** [comment]
I met that problem too. It was because the User-Agent http header did not contain the "Version xx.x" string.

You can fix it either by changing the UIWebView http header value (I did not suceed to do that) or either by changing the test in raphaeljs (way much easier).

**Comment by leonfu** [comment]
Have u tried to run in your own app with uiwebview, NOT mobile safari. It will crash your app for ios 3.1 below

**Comment by Chralu** [comment]
Yes my app uses a UIWebView. It works for iOS>=3.0. But I use raphael and not g.raphael. As the error is the same I think your problem is in raphael too. Look for the regexp containing 'Version' on UserAgent.

**Comment by Chralu** [comment]
A good way to know where that bug comes from is surrounding your code with a 
"try{...}catch(e){alert(e);}"
